### PR TITLE
Jetpack Onboarding: Introduce generic enter animations on all steps

### PR DESCRIPTION
**Pull request by tyxla**
This PR introduces a very generic enter animation on all Jetpack Onboarding step - fading in with scaling from 50% to 100%. It's pretty basic, but should do the job as a first iteration.

To test:
* Checkout this branch.
* Go to http://calypso.localhost:3000/jetpack/onboarding/homepage/site-title
* Skip consistently to reach the last Summary step, and in the meantime verify that you can see the entry animation when each of the steps is displayed.

### CODE REVIEW

File: client/jetpack-onboarding/style.scss [hunk 1]
```diff
@@ -1,4 +1,8 @@
 .jetpack-onboarding {
+	.steps__main {
+		animation: appearWithScaling .3s ease-in-out;
```
Comment (AnnaMag) [inline client/jetpack-onboarding/style.scss]: Nit: a slower rate (e.g. `.8s`) seems easier on the eyes to me. Those blocks are large and fast changes create a flickery effect.

**Comment by MichaelArestad** [comment]
Can we change the scale from 50% to something more subtle like 90%?

File: client/jetpack-onboarding/style.scss (hunk 1 above)
Comment (tyxla) [reply client/jetpack-onboarding/style.scss]: Tried that, but I think Michael's suggestion worked really well in mitigating the flickery effect you're talking about (increasing initial scale from 0.5 to 0.9).

**Comment by tyxla** [comment]
> Can we change the scale from 50% to something more subtle like 90%?

Brilliant, thanks Michael 👍

**Pull request merged**